charts/misc-apps/README.md

```diff
@@ -0,0 +1,47 @@
+# misc-apps
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+
+Argo CD app-of-apps config for miscellaneous small tools
+
+**Homepage:** <https://github.com/adfinis-sygroup/helm-charts/tree/master/charts/misc-apps>
+
+## Maintainers
+This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk_kwd=helm-charts).
+
+## Source Code
+
+* <https://github.com/adfinis-sygroup/helm-charts>
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://charts.adfinis.com | argoconfig | 0.6.0 |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| downscaler | object | see [values.yaml](./values.yaml) | [kube-downscaler](https://github.com/hjacobs/kube-downscaler) |
```

Comment:
I changed this so the link to `values.yaml` isn't first in line anymore. I also add more more values so the README.md now reflects everything in values.yaml.

Reply:
It is definitely better now.
I'm still not 100% happy with it (e.g. it is still not clear that it would work to add some other values that would indeed be used and have effects, like it is done for `downscaler.project` in the example.)
However, this is in the documentation, so clearly a non-functional issue. Also, I do not have any good idea how to really improve it, so I think it should be fine for now.